qa/tasks/ceph.py

```diff
@@ -793,6 +793,16 @@ def cluster(ctx, config):
                     '--monmap', monmap_path,
                 ],
             )
+            mnt_point = DATA_PATH.format(
```

Comment:
@tchaikov Need review here. In luminous, chown was not being run on osd mount path.